test/utils/describeConformanceV5.js

```diff
@@ -28,13 +30,94 @@ function testComponentsProp(element, getOptions) {
   });
 }
 
+/**
+ * Material-UI theme has a components section that allows specifying default props, overrides and variants
+ * Components from @inheritComponent
+ * @param {React.ReactElement} element
+ * @param {() => ConformanceOptions} getOptions
+ */
+function testThemeComponents(element, getOptions) {
+  const render = createClientRender();
+
+  describe('theme: components', () => {
+    it("respect theme's defaultProps", () => {
+      const { muiName, testThemeComponentsDefaultPropWith: testProp = 'id' } = getOptions();
+      const theme = createMuiTheme({
+        components: {
+          [muiName]: {
+            defaultProps: {
+              [testProp]: 'testProp',
+            },
+          },
+        },
+      });
+
+      const { container } = render(<ThemeProvider theme={theme}>{element}</ThemeProvider>);
+
+      expect(container.firstChild[testProp]).to.equal('testProp');
```

Comment:
```suggestion
      expect(container.firstChild).to.have.attribute(testProp, 'testProp');
```

Otherwise you're testing the **property**. However, props are most of the time just attributes.